_posts/2022-03-02-getting-started-project-flotta.markdown

```diff
@@ -45,14 +45,12 @@ $ kubectl wait --for=condition=Ready pods --all -n openshift-ingress --timeout=6
 
 [Noobaa](https://noobaa.github.io/) is data service that provides S3 object-store interface. Flotta's Edge API enables
 edge devices to send application data to S3 endpoint, into a bucket claim created automatically for each device at
-registration time.
+registration time. By default, Object Bucket Claim auto-creation is disabled. To enable it, follow the instructions [below](#enable-object-bucket-claim-auto-creation).
```

Comment:
nit: This is the first occurrence of Object Bucket Claim, what do you think to define the acronym here? (i.e. Object Bucket Claim (OBC) )

In addition, I'd like to suggest to use italic to keep the style consistent .

Reply:
I'm not sure it's clear to a first time contributor that Noobaa is needed if and only if OBC is enabled. Perhaps an explicit sentence such as "To use noobaa you must enable OBC" might help understanding. Any thoughts on this?